minecraft_servers/letsgrief/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "server_name": "letsgrief",
+    "nice_name": "LetsGrief",
+    "direct_ip": "mc.letsgrief.net",
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://web.letsgrief.net",
+        "web_support": "https://web.letsgrief.net/support",

```

Comment:
Can't be hidden behind login; either create support form or faq

Reply:
Support without login is impossible. Witout login we cant provide support because then were not able to contact the member. Then i must remove this